test/GitIgnoreTest.php

```diff
@@ -91,13 +119,22 @@ public function testAddEntryDoesNotAddFileOrDirectoryIfItIsInsideAnIgnoredDirect
 
     public function testIgnoreDirectoriesAreNotWrittenIfNoEntriesAreAdded()
```

Comment:
Yeah so this test shows that the feature was intentional. My thoughts were that why would you add these folders if you are not actually adding anything else? I'm not against the change, but if we implement then we need to remove this test but it's not doing as it actually describes now.